client/utilities/productMovePreview.ts

```diff
@@ -0,0 +1,22 @@
+// These functions are client side versions of logic performed in the product-move-api to render preview details
+// They are duplicated here to avoid making many api calls if a user selects different amounts that require preview
+
+/**
+ * calculate the amount a user will pay at the time they switch product
+ */
+export function calculateAmountPayableToday(
+	userChosenAmount: number,
+	contributionRefundAmount: number,
+): number {
+	// contributionRefundAmount will be a negative number (eg if the refund is £5 the param will be -5)
+	return userChosenAmount - -1 * contributionRefundAmount;
```

Comment:
Is -1 * contributionRefundAmount just doing the same as `Math.abs()`? that would be easier to read for me, it took me a while to parse this!

Reply:
yes I agree this is ugly to look at! I think prettier reformatted this to remove brackets I put around the multiplication...

Should it be `Math.abs` ... I guess I wasn't sure if there was an edgecase where the refundAmount would actually be a positive number. If we can confirm this won't happen then let's change to using absolute value

Reply:
^ or alternatively just add the two together and let the comment explain why

Reply:
Refund amount will never be positive unless there's a bug somewhere else, either way we want to end up with a positive number so I think abs is the way to go.